python/pyspark/sql/window.py

```diff
@@ -196,11 +208,23 @@ def rangeBetween(start: int, end: int) -> "WindowSpec":
         --------
         >>> from pyspark.sql import Window
         >>> from pyspark.sql import functions as func
-        >>> from pyspark.sql import SQLContext
-        >>> sc = SparkContext.getOrCreate()
-        >>> sqlContext = SQLContext(sc)
-        >>> tup = [(1, "a"), (1, "a"), (2, "a"), (1, "b"), (2, "b"), (3, "b")]
-        >>> df = sqlContext.createDataFrame(tup, ["id", "category"])
+        >>> df = spark.createDataFrame(
```

Comment:
you will probably have to define `spark` in the bottom.

Reply:
Yeah. without define `spark`, pyspark tests failed. Updated in https://github.com/apache/spark/pull/37450/commits/34623307de7809c76cbcfe864d2a3b4df98a61c5